From d437ecc0306c8b4b1c9737792963da4f7f4e4234 Mon Sep 17 00:00:00 2001
From: Sebastian Geiger <sbastig@gmx.net>
Date: Sun, 11 Feb 2018 15:27:29 +0100
Subject: [PATCH] Install appstream metadata in new location

Installing metadata into ${datadir}/appdata is deprecated. The new location
is ${datadir}/metainfo.

See also https://lintian.debian.org/tags/appstream-metadata-in-legacy-location.html
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 73a8e95..8d935e8 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -39,7 +39,7 @@ MAINTAINERCLEANFILES = \
 Applicationsdir = ${datadir}/applications
 Applications_DATA = tilda.desktop
 
-Appdatadir = ${datadir}/appdata
+Appdatadir = ${datadir}/metainfo
 Appdata_DATA = tilda.appdata.xml
 
 Pixmapsdir = ${datadir}/pixmaps
